src/main/scala/strawman/collection/mutable/ArrayDeque.scala

```diff
@@ -0,0 +1,292 @@
+package scala
+package collection.mutable
```

Comment:
I need `.sizeHintIfCheap` which is package visible to `scala.collection.mutable` only.